documentation/user_docs/docs/manual/Parallel.rst

```diff
@@ -0,0 +1,130 @@
+##########################
+Running Horace in Parallel
+##########################
+
+
+Controlling MPI
+===============
+
+Certain operations in Horace are parallelised with an MPI algorithm. These can be enabled in one go using:
+
+::
+
+   hpc('on')
+
+In order to manually enable/disable different parallel components it is possible, these are managed through the ``hpc_config``.
```

Comment:
"it is possible" looks wrong.
Also does manually mean not via hpc on?
Simpler to say "Alternatively different parallel components can be enabled/disabled separately through hpc_config"